packages/api/src/utils/format-money.ts

```diff
@@ -0,0 +1,23 @@
+import { useAuthorize } from '../hooks';
+
+type TCurrency = NonNullable<ReturnType<typeof useAuthorize>['data']['currency']>;
+type TPreferredLanguage = ReturnType<typeof useAuthorize>['data']['preferred_language'];
+
+export const formatMoney = (
```

Comment:
Can we have a different name, since this might conflict with the one we already have in shared. 
Here are some examples 😸 

formatCurrency
convertToCurrencyFormat
displayMoney
moneyFormatter
formatAsMoney
currencyFormatter
convertMoneyToString
moneyToString
formatMoneyValue
displayCurrency

Reply:
ditto

Reply:
Have renamed